python-package/lightgbm/callback.py

```diff
@@ -438,6 +436,18 @@ def __call__(self, env: CallbackEnv) -> None:
             self._final_iteration_check(env, eval_name_splitted, i)
 
 
+def _should_enable_early_stopping(stopping_rounds: Any) -> bool:
+    """Check if early stopping should be activated.
+
+    This function will evaluate to True if the early stopping callback should be
+    activated (i.e. stopping_rounds > 0).  It also provides an informative error if the
+    type is not int.
+    """
+    if not isinstance(stopping_rounds, int):
+        raise TypeError(f"early_stopping_round should be an integer. Got " f"{type(stopping_rounds).__name__}")
```

Comment:
```suggestion
        raise TypeError(f"early_stopping_round should be an integer. Got {type(stopping_rounds).__name__}")
```

I'm guessing `black` combined these like this. I think we should remove the extra quotes and `f` modifier.